### ISSUE TITLE
Accessibility Issues in Edit Article Page

### ISSUE THREAD

**Issue by ajitava-git** [qxf2/newsletter_automation#227]
The below-mentioned issues are classified as critical, serious, and moderate.

**Critical:**

1. "aria-hidden='true' must not be present on the document body" 'helpUrl': [Axe Rules | Deque University | Deque Systems](https://dequeuniversity.com/rules/axe/3.1/aria-hidden-body?application=axeAPI)

2. every form element has a label 'helpUrl': '[Axe Rules | Deque University | Deque Systems](https://dequeuniversity.com/rules/axe/3.1/label?application=axeAPI) '

**Serious:**

1. The contrast between foreground and background colors should meet WCAG 2 AA contrast ratio thresholds. Elements must have sufficient color contrast.
'bgColor': None, 'contrastRatio': 0, 'expectedContrastRatio': '4.5:1', 'fgColor': '#212529' 'helpUrl': '[Axe Rules | Deque University | Deque Systems](https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=axeAPI) '

2. Document has a non-empty <title> element

**Moderate:**

Page doesn't have banner landmark, contentinfo landmark 'helpUrl': '[Axe Rules | Deque University | Deque Systems](https://dequeuniversity.com/rules/axe/3.1/landmark-no-duplicate-banner?application=axeAPI) '

**Comment by ajitava-git** [comment]
The accessibility tests run fine now in CI.
